types/tests/providers.test.ts

```diff
@@ -136,7 +137,7 @@ Providers.IdentityServer4({
 Providers.Discord({
   clientId: "foo123",
   clientSecret: "bar123",
-  scope: "identify",
+  authorization: { params: { scope: "identify" } },
```

Comment:
I'm using beta.2 and this doesn't seem to work, whatever I do the scope in the URL remains as `scope=identify%20email`.

I do not want to know user's their email.

Reply:
Please open an issue/discussion and attach a reproduction. This is not the right place. You are commenting on a test file. 👀